packages/SwingSet/src/kernel/vatManager.js

```diff
@@ -116,7 +111,9 @@ export default function makeVatManager(
     insistCapData(args);
     // TODO: disable send-to-self for now, qv issue #43
     const target = mapVatSlotToKernelSlot(targetSlot);
-    kdebug(`syscall[${vatID}].send(vat:${targetSlot}=ker:${target}).${method}`);
+    const argList = legibilizeMessageArgs(args).join(', ');
```

Comment:
is there an easy way to avoid the perf hit when kdebug is disabled?

I'm ok with landing this as-is, but I'd like a ticket to remind us to speed it up in the next few weeks.

Reply:
Filed as #925